Adapters/Vungle/ISVungleAdapter/ISVungleAdapter.h

```diff
@@ -5,7 +5,7 @@
 #import <Foundation/Foundation.h>
 #import "IronSource/ISBaseAdapter+Internal.h"
 
-static NSString * const VungleAdapterVersion = @"4.3.22";
+static NSString * const VungleAdapterVersion = @"5.0.0";
```

Comment:
@YueVungle  Doesn't the versioning map to SDK version (like with other adapters)? e.g. 7.x

Reply:
@shiv-vungle I have checked the commit history. The IronSource Vungle Adapter versions are not mapping to our SDK versions. Thanks :)